src/estimator/mean-squared-error.ts

```diff
@@ -22,26 +35,30 @@ class MeanSquaredError {
       },
       immutable: true,
     });
+
+    // Need more info about what value1 and value2 is.
     this.addAbsolute = makeKernel(
-      function (value1, value2) {
+      function (value1: any, value2: any) {
```

Comment:
I believe:
```
value1: number[]
value2: number[][]
```

Reply:
Woah! I had made it number[] and number[][], idk how it disappeared.

Reply:
But I want to know what they are instead of just the types. This will be helpful for *future humans* as you call them :laughing:.

Reply:
>I want to know what they are instead of just the types. 

What do you mean?

Reply:
The variable names arent quite obvious. Also it isnt obv why the second one is [][] and first is []. My intention is to add some comments/jsdoc for future humans.

Reply:
Its summing a matrix of errors into the previous error: https://github.com/BrainJS/brain.js/blob/6ddd74c3f3acf0ba778766e9ec6a27014bc67272/src/recurrent.js#L324

I did my best with names, and to try and keep this on the GPU as much as possible.  If there is a better naming convention, I'm open to it.

The usage is here: https://github.com/BrainJS/brain.js/blob/6ddd74c3f3acf0ba778766e9ec6a27014bc67272/src/recurrent.js#L324 which imitates the existing CPU counterpart: https://github.com/BrainJS/brain.js/blob/6f661e442682209b235773cad31b19931593b27b/src/recurrent/rnn-time-step.js#L663

Reply:
Ty. I'll update it :)